ui-tests/cypress/support/pageObjects/app/DeleteAdminPage.js

```diff
@@ -21,11 +21,22 @@ export default class DeleteAdminPage extends AdminContent {
     cy.wait('@deleteCategoryPageLoadingPOST');
   }
 
+  //FIXME AdminConsole is not built on REST APIs
+  static openDeleteContentTypePage(button, code) {
+    cy.contentTypesAdminConsoleController().then(controller => controller.intercept({ method: 'GET' }, 'deleteContentTypePageLoadingGET', `/trashEntityType.action?entityManagerName=jacmsContentManager&entityTypeCode=${code}`));
+    cy.get(button).click();
+    cy.wait('@deleteContentTypePageLoadingGET');
+  }
+
   getForm() {
     return this.getContents()
                .children(this.form);
   }
 
+  setForm(newForm) {
```

Comment:
how should this dynamic form work?

Reply:
The id of the form when accessing the delete admin page from the content types is different, so when the clickDelete in TypesPage is called, the setForm method is called to change the identifier that the getForm method looks for